src/packages.builds

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="14.0" DefaultTargets="BuildAndTest" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
BTW we don't need xmlns on any MSBuild files anymore as far as I know.

Reply:
right, please just use the Project tag and the copyright line as you already did in an arcade package.